src/WebJobs.Script/Description/Worker/WorkerFunctionDescriptorProvider.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.ObjectModel;
+using System.Threading.Tasks.Dataflow;
+using Microsoft.Azure.WebJobs.Script.Binding;
+using Microsoft.Azure.WebJobs.Script.Description.Script;
+using Microsoft.Azure.WebJobs.Script.Dispatch;
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    internal class WorkerFunctionDescriptorProvider : FunctionDescriptorProvider
```

Comment:
I keep wondering if we shouldn't rename the "Worker" terminology to "Rpc" or something. E.g.: RpcFunctionDescriptionProvider, RpcFunctionInvoker, etc.

Reply:
I agree it needs to be renamed. Really dislike RPC for this, though.